src/site/stages/build/process-cms-exports/schemas/output/node-health_services_listing.js

```diff
@@ -22,7 +25,14 @@ module.exports = {
     fieldMetaTitle: { type: 'string' },
     fieldOffice: {
       type: 'object',
-      properties: { entity: { $ref: 'output/node-health_care_region_page' } },
+      properties: {
+        /* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
I guess the react-hooks rule just looks for any function starting with `use`? That's lame. 😆

Reply:
Fixed by #14754